core/domain/voiceover_domain.py

```diff
@@ -51,9 +51,9 @@ class EntityVoiceovers:
         entity_version: str. The version of the corresponding entity.
         language_accent_code: str. The language accent code in which the
             voiceover is stored.
-        voiceovers: dict(str, dict(str, VoiceoverDict)). A dict representing
-            content IDs as keys and a nested dict as values. Each nested dict
-            contains VoiceoverType as keys and VoiceoverDict
+        voiceovers: dict(str, dict(VoiceoverType, VoiceoverDict)). A dict
+            containing content IDs as keys and nested dict as values.
```

Comment:
See the earlier comment: this should be "and nested dicts".

Reply:
Updated